ztools/x509/pkix/pkix.go

```diff
@@ -208,3 +208,17 @@ type RevokedCertificate struct {
 	RevocationTime time.Time
 	Extensions     []Extension `asn1:"optional"`
 }
+
+// OtherName represents the ASN.1 structure of the same name. See RFC
+// 5280, section 4.2.1.6.
+type OtherName struct{
```

Comment:
I did try adding a asn1:"tag:1,explicit", it appears it has no effect when parsing into an asn1.RawValue. The Value field here _is_ the tag, with the contents stored in the Bytes field.